docs/index.md

```diff
@@ -1,13 +1,94 @@
-El Collectooorr aims to autonomously create collections of NFTs by minting them as they drop on Art Blocks. It uses complex logic to decide which mints to participate in and what prices to bid at, so users don’t have to. The service is fueled by the community of users who fund collections. Each collection is designed to have its own unique ERC-20 token, for users to vote and collectively decide on what happens to the collection.
+![MintKit](images/el-collectooorr.png){ align=left width="150" }
+The MintKit is a toolkit to build services with minting capabilities. For example, El Collectooorr service, which is based on the MintKit, aims to autonomously create collections of NFTs by minting them as they drop on Art Blocks. It uses complex logic to decide which mints to participate in and what prices to bid at, so users don’t have to. The service is fueled by the community of users who fund collections. Each collection is designed to have its own unique ERC-20 token, for users to vote and collectively decide on what happens to the collection.
 
+## Demo
 
-##Run the code
+!!! warning "Important"
 
-!!! info
-	This section will be added soon.
+	This section is under active development - please report issues in the [Autonolas Discord](https://discord.com/invite/z2PT65jKqQ).
```

Comment:
Please make sure the link is non-expiring

Reply:
This is the usual discord link, which I believe is not expiring. In any case, I'll add open a task for the Server Owner on the related document to double-check it.